cmd/upgrade/operatorroles/cmd.go

```diff
@@ -204,37 +206,38 @@ func run(cmd *cobra.Command, argv []string) error {
 	}
 
 	if isOperatorPolicyUpgradeNeeded {
-		err = upgradeOperatorPolicies(mode, r, prefix, isAccountRoleUpgradeNeed,
-			policies, env, defaultPolicyVersion, credRequests, cluster, policyPath)
+		err = upgradeOperatorPolicies(
```

Comment:
Should we stick to the current convention in the project and have all the parameters in the same line? 
Like the convention in CS.